src/tls13.c

```diff
@@ -850,6 +850,26 @@ static int DeriveTrafficSecret(WOLFSSL* ssl, byte* secret)
                      ssl->specs.mac_algorithm, 0);
 }
 
+
+static inline int Tls13_HKDF_Extract(WOLFSSL *ssl, byte* prk, const byte* salt, int saltLen,
+                                 byte* ikm, int ikmLen, int digest)
+{
+    int ret;
+#ifdef HAVE_PK_CALLBACKS
+        void *cb_ctx = ssl->HkdfExtractCtx;
+        CallbackHKDFExtract cb = ssl->ctx->HkdfExtractCb;
+        if ((cb != NULL) && (saltLen > 0)) {
```

Comment:
Why can the callback only be called on `saltLen > 0`?

Reply:
This was in fact a requirement of one specific IoT-Safe caller. I agree it should not be here. Fixed.